src/harness/testcases/configs/test_WINNF_FT_S_GRA_5/default.config

```diff
@@ -29,11 +29,11 @@
                 "highFrequency": 3630000000
               }
             },
-            "id": "SAMPLE_ID_29642",
-            "grantExpireTime": "2020-03-02T17:29:37Z"
+            "id": "SAMPLE_ID_22957",
+            "grantExpireTime": "2020-03-07T15:58:23Z"
           }
         ],
-        "id": "cbsd/test_fcc_id_a/d7a9fe1be84243ebdd50c1359cf0630c3d273350",
+        "id": "cbsd/cbsd/test_fcc_id_a/d7a9fe1be84243ebdd50c1359cf0630c3d273350",
```

Comment:
Hmm, I just noticed that this is getting prepended twice. Probably a small bug in `generateCbsdRecords()`.

Reply:
Fixed   the bug in generateCbsdRecords()